examples/cpp_execution/README.md

```diff
@@ -0,0 +1,31 @@
+
+# Text Generation using CPP Inference
+
+## Overview
+This example demonstrates how to execute a model on AI 100 using Efficient Transformers and C++ APIs. The Efficient Transformers library is utilized for transforming and compiling the model, while the QPC is executed using C++ APIs. It is tested on both x86 and ARM platform.
+
+## Prerequisite
+1. PyBind11
+2. Cpp17 or above
```

Comment:
is this clear enough, it's better to include what gcc version you have tested on. You can add a line saying the below readme file is validated with cpp version ** and gcc version **.